src/Model/Message.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace App\Model;
+
+use Symfony\Component\Validator\Constraints as Assert;
+
+class Message
+{
+    public const TYPE_SMS = 'sms';
+    public const TYPE_EMAIL = 'email';
+    /**
+     * @Assert\NotBlank(message="body cannot be empty", payload="101")
+     */
+    public string $body = '';
+
+    /**
+     * @Assert\NotBlank(message="type cannot be blank", payload="102")
+     */
+    public string $type = self::TYPE_EMAIL;
```

Comment:
This is a potential bug. If we want to define a default service, it should be done in corresponding services and clearly stated, what we are defining default service